scripts/update-azure-blob-storage/main.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * This file:
+ * - Updates dso-toolkit.nl/versions.json with all the deployed releases (tags and branches) in the Azure Blob Storage container.
```

Comment:
Hoe werd dit voorheen geregeld? En waarom is dit nu nodig?

Reply:
Vroeger werd dit in https://github.com/dso-toolkit/dso-toolkit/pull/2683/files#diff-30883d1df2fc62f59d66255a70dc05f60d3d4ef4d3957e6e3e38e62f6e471bd0L38 geregeld:

Er werd een SSH verbinding gestart waarna er een NPM script op de host werd gedraaid. Dit script voerde uit wat er nu in `main.ts` staat.

Ik zal het oude script in een comment van dit PR plakken :+1:

Reply:
Eric zegt ✅